pkg/kor/kor.go

```diff
@@ -199,6 +199,7 @@ func formatOutputForResource(resource string, resources map[string][]string, opt
 			index++
 		}
 	}
+	table.Append([]string{})
```

Comment:
This doesn't seem to affect it.

Reply:
Yeah, I tested with a old binary I built so thought it did.

Reply:
I have pushed up a change that matches the expected spacing.